lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -24,6 +24,9 @@ bool LLDBMemoryReader::queryDataLayout(DataLayoutQueryType type, void *inBuffer,
       return false;
     // The mask returned by the process masks out the non-addressable bits.
     uint64_t mask_pattern = ~ptrauth_mask;
+    // Keep the lldb file address bit, so we discern between real and 
+    // lldb's vertiual address space.
```

Comment:
virtual

Reply:
How about: LLDBMemoryReader

Reply:
How about: `LLDBMemoryReader sets LLDB_FILE_ADDRESS_BIT to distinguish process addresses and file addresses that point into a reflection section on disk. Setting the bit in the mask ensures it isn't accidentally cleared by ptrauth stripping.`